data/countries/CN.yaml

```diff
@@ -39,38 +56,66 @@ holidays:
         name:
           en: National Day
           zh: 国庆节
-      10-02:
+      10-31:
         name:
-          en: National Day
-          zh: 国庆节
-      10-03:
+          en: Halloween
+          zh: 万圣节
+      12-24:
         name:
-          en: National Day
-          zh: 国庆节
+          en: Christmas Eve
+          zh: 平安夜
+      12-25:
+        name:
+          en: Christmas
+          zh: 圣诞节
       chinese 01-0-00: # Spring Festival Eve
         name:
-          en: Spring Festival
-          zh: 春节
+          en: New Year's Eve
+          zh: 除夕
       chinese 01-0-01: # 1st day of 1st lunar month
         name:
           en: Spring Festival
           zh: 春节
```

Comment:
According to https://www.gov.cn/gongbao/2024/issue_11726/202411/content_6989767.html Spring Festival is over a period of 7 days. Shouldn't we reflect this here?